### PR TITLE
Made solo winner games not have any DIAS members in the results.

### PR DESCRIPTION
**Pull request by zond**
- Made it happen in game finish code.
- Added endpoint to fix it ONCE AND FOR ALL.
- Added test to make sure it works.

### CODE REVIEW

**Comment by zond** [comment]
@tttppp Please take a look at this, it should solve https://github.com/zond/diplicity/issues/111

File: diptest/phase_test.go [hunk 1]
```diff
@@ -92,6 +145,353 @@ func TestStartGame(t *testing.T) {
...
+	},
+	{},
+}
+
+func TestSoloEnding(t *testing.T) {
+	withStartedGameOptsAndOrders(nil, russianSoloOrders, func() {
+		orderSet{
+			{
+				nat:  "England",
+				dias: true,
```
Comment (tttppp) [inline diptest/phase_test.go]: Cowardly England

File: diptest/phase_test.go [hunk 2]
```diff
@@ -92,6 +145,353 @@ func TestStartGame(t *testing.T) {
...
+			},
+		}.execute(len(russianSoloOrders) + 1)
+		game := startedGameEnvs[0].GetRoute("Game.Load").RouteParams("id", startedGameID).Success()
+		game.AssertBoolEq(true, "Properties", "Finished")
+		game.Follow("game-result", "Links").Success().AssertNil("Properties", "DIASMembers").AssertEq("Russia", "Properties", "SoloWinnerMember")
```
Comment (tttppp) [inline diptest/phase_test.go]: Is there a list of "Losers" too?  If so then is it worth checking England is present?

Comment (zond) [reply diptest/phase_test.go]: No loser list, it's just assumed that if someone wins, every one else loses :D

**Pull request merged**